ExcelToCWMSTests/ProcessDataTableTests.cs

```diff
@@ -33,7 +33,7 @@ public void GetTSStartTimeTest()
             DateTime t1 = DateTime.Parse("2021-03-01");
             DateTime t2 = DateTime.Parse("2021-03-03");
 
-            var TimeSeriesArray = ProcessDataTable.GetTimeSeriesFromExcel("input.xlsx", "import", t1, t2, new TimeSpan(0, 0, 0));
+            var TimeSeriesArray = ProcessDataTable.GetTimeSeriesFromExcel("input.xlsx", "import", t1, t2, TimeZoneInfo.Local);
             Console.WriteLine( "Start Time: "+ TimeSeriesArray[0].getTSStartTime());
```

Comment:
I think it would be more beautiful , and c# like, if you could write:

TimeSeriesArray[0].StartDateTime   (property)
instead of 
TimeSeriesArray[0].getTSStartTime()